packages/test/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "type": "module",
   "engines": {
-    "node": ">=10 <13 || ^13.7 || >=14"
+    "node": "^13.7 || >=14"
```

Comment:
the OP says "requires node 12" but this drops it. do you mean `>= 12 || ^13.7 || >= 14`?